### ISSUE TITLE
Numerical keyboard NOT telephone?

### ISSUE THREAD

**Issue by FractalMind** [nbfontana/ngx-currency#136]
I know there's an issue about the numerical keyboard and how you should use type="tel" to pop the telephone keyboard

But for a CURRENCY mask, the numerical keyboard is way more appropriate. 

Is there any workaround?

Thank you

**Comment by Jeremy-WinonaIT** [comment]
I completely agree. It's seems like it's working as a side-effect and not as an intended behavior
